### PR TITLE
Jorge/syntactic typos

### PR DESCRIPTION
**Pull request by jorgefandinno**
Some typos corrected:
 - abbreviations should be followed by "." instead of ","
- names of people should be separeted by "and" instead of ","

### CODE REVIEW

File: krr.bib [hunk 1]
```diff
@@ -354,6 +354,7 @@ @InProceedings{albrlepe02a
 
 @InProceedings{alcadofuleperiveza17a,
   author =	 "M. Alviano and F. Calimeri and C. Dodaro and D. Fusc{\`a} and N. Leone and S. Perri and F. Ricca and P. Veltri and J. Zangari",
+
```
Comment (tortinator) [inline krr.bib]: Why insert a newline?
At least it appears to me like this..
Chao! -t

File: krr.bib [hunk 2]
```diff
@@ -917,6 +918,7 @@ @Misc{asparagus
 
 @Misc{aspcore2,
   author =	 "F. Calimeri and W. Faber and M. Gebser and G. Ianni and R. Kaminski and T. Krennwallner and N. Leone and F. Ricca and T. Schaub",
+
```
Comment (tortinator) [inline krr.bib]: ditto,
also for below

File: krr.bib [hunk 3]
```diff
@@ -3292,7 +3299,8 @@ @InProceedings{brass91
   title =	 "Deduction with Supernormal Defaults",
   booktitle =	 "Nonmonotonic and Inductive Logic",
   year =	 1991,
-  editor =	 "G. Brewka, K. Jantke, P. Schmitt",
+  editor =	 "G. Brewka and K. Jantke and P. Schmitt",
```
Comment (tortinator) [inline krr.bib]: Thx!

File: krr.bib [hunk 4]
```diff
@@ -16326,7 +16347,8 @@ @InCollection{reiter84a
   booktitle =	 "On conceptual modeling: Perspectives from Artificial Intelligence, Datbases and Programming Languages",
   publisher =	 springer,
   year =	 1984,
-  editor =	 "M. Brodie, J. Myopoulos, J.W. Schmidt",
+  editor =	 "M. Brodie and J. Myopoulos and J.W. Schmidt",
```
Comment (tortinator) [inline krr.bib]: Thx!

File: krr.bib (hunk 1 above)
Comment (jorgefandinno) [reply krr.bib]: That was unintended

File: krr.bib (hunk 2 above)
Comment (jorgefandinno) [reply krr.bib]: Is this solved now?

**Comment by tortinator** [comment]
I address many of the changes, an dmerged them into master.

Just pull the master into your branch and see whether anything is still open..?

**Pull request closed without merging**